package.json

```diff
@@ -34,53 +34,53 @@
     "canvas": "^2.8.0",
     "cheerio": "1.0.0-rc.10",
     "copy-to-clipboard": "^3.3.1",
-    "date-fns": "^2.23.0",
+    "date-fns": "^2.24.0",
     "fast-xml-parser": "^3.20.0",
-    "gatsby": "3.13.0",
+    "gatsby": "3.14.0",
     "gatsby-background-image": "^1.5.3",
     "gatsby-image": "^3.11.0",
-    "gatsby-plugin-canonical-urls": "^3.13.0",
-    "gatsby-plugin-catch-links": "^3.13.0",
-    "gatsby-plugin-feed": "^3.13.0",
-    "gatsby-plugin-image": "^1.13.0",
-    "gatsby-plugin-manifest": "3.13.0",
+    "gatsby-plugin-canonical-urls": "^3.14.0",
+    "gatsby-plugin-catch-links": "^3.14.0",
+    "gatsby-plugin-feed": "^3.14.0",
+    "gatsby-plugin-image": "^1.14.0",
+    "gatsby-plugin-manifest": "3.14.0",
     "gatsby-plugin-netlify": "^3.14.0",
-    "gatsby-plugin-react-helmet": "4.13.0",
+    "gatsby-plugin-react-helmet": "4.14.0",
     "gatsby-plugin-react-i18next": "^1.1.1",
     "gatsby-plugin-robots-txt": "^1.6.10",
-    "gatsby-plugin-sass": "^4.13.0",
-    "gatsby-plugin-sharp": "3.13.0",
-    "gatsby-plugin-sitemap": "^4.9.0",
+    "gatsby-plugin-sass": "^4.14.0",
+    "gatsby-plugin-sharp": "3.14.0",
+    "gatsby-plugin-sitemap": "^4.10.0",
     "gatsby-plugin-smoothscroll": "^1.2.0",
-    "gatsby-plugin-styled-components": "^4.13.0",
-    "gatsby-plugin-typescript": "3.13.0",
-    "gatsby-remark-copy-linked-files": "^4.10.0",
+    "gatsby-plugin-styled-components": "^4.14.0",
+    "gatsby-plugin-typescript": "3.14.0",
+    "gatsby-remark-copy-linked-files": "^4.11.0",
     "gatsby-remark-external-links": "^0.0.4",
-    "gatsby-remark-images": "^5.10.0",
-    "gatsby-remark-prismjs": "^5.10.0",
+    "gatsby-remark-images": "^5.11.0",
+    "gatsby-remark-prismjs": "^5.11.0",
     "gatsby-remark-reading-time": "^1.1.0",
-    "gatsby-source-filesystem": "3.13.0",
-    "gatsby-transformer-json": "^3.13.0",
-    "gatsby-transformer-remark": "^4.10.0",
-    "gatsby-transformer-sharp": "3.13.0",
+    "gatsby-source-filesystem": "3.14.0",
+    "gatsby-transformer-json": "^3.14.0",
+    "gatsby-transformer-remark": "^4.11.0",
+    "gatsby-transformer-sharp": "3.14.0",
     "html-entities": "^2.3.2",
     "i18next": "^20.6.1",
     "leaflet": "^1.7.1",
     "modern-normalize": "1.1.0",
-    "node-fetch": "2.6.1",
+    "node-fetch": "3.0.0",
```

Comment:
@feedm3 any options to configure the bump? It's just forcing any package to upgrade ignoring pinned versions (like I did for this problematic package) or for the actual version ranges itself.

Reply:
The version bump is currently super simple, as it only does a `yarn upgrade --latest`. I did some (quick) research and couldn't find an easy way to exclude some dependencies from upgrading. 

With version 3 `node-fetch` only support ESM (`import ...`) and no `require` anymore: 
- https://github.com/node-fetch/node-fetch/blob/main/docs/v3-UPGRADE-GUIDE.md#converted-to-es-module
- https://github.com/node-fetch/node-fetch/issues/1279

They propose an async import, but this doesn't work either. Probably related to the way webpack is configured by Gatsby.

We use `node-fetch` only in one place: To do a single `GET` request to Personio. I think it's best to replace `node-fetch` with some other dependency like axios. I will updated this PR.

Reply:
Feel free to merge this PR if you are fine with it.